examples/visual-tests/amp-story/amp-story-live-story.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const {verifySelectorsVisible, verifySelectorsInvisible} = require('../../../build-system/tasks/visual-diff/helpers');
+
+module.exports = {
+  'Inserting a new page should show a notification in the last page.': async (page, name) => {
+    const screen = page.touchscreen;
+    await screen.tap(200, 240);
```

Comment:
Can we find this line that works on both mobile and desktop so we can save one file of repeated code?

Reply:
Done.